container_build.sh

```diff
@@ -116,12 +116,24 @@ process_all_targets() {
   done
 }
 
+process_multi_arch_targets() {
+  for i in $(cat ./multi-arch-targets.list); do
+    build_multi_arch "$i"
+  done
+}
+
+build_multi_arch() {
```

Comment:
Can this be simplified by passing in "farm" to build_arch()
and then doing `podman $2 build ...`

Reply:
BTW This is a good idea, and potentially useful for automated builds.

Reply:
Let me see.

`podman farm` does not like a tag on the image.  So, this might be more complex.

Reply:
Ah, but the script does not explicitly add the `latest` tag either.  So this might work.

I'll tinker with it.

Reply:
@rhatdan I think I succeeded in simplifying based on your request.